projects/train/train/data/supervised/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .frequency_domain import FrequencyDomainSupervisedAframeDataset
```

Comment:
Need to import `SpectrogramSuperivsedAframeDataset`. Also, I think we should either leave the `FrequencyDomainSupervisedAframeDataset` name as-is, or remove `Supervised` from the other class names to maintain consistency. I'm good with either option.